lib/helpers.js

```diff
@@ -57,9 +57,11 @@ exports.prefixedHash = x => {
 
 exports.sha256 = x => crypto.createHash('sha256').update(x).digest('base64');
 
-exports.generateNonce = async ({entryStyle = 'human'} = {}) => {
+exports.generateNonce = async ({
+  entryStyle = 'human', nonceLength = 9
```

Comment:
Hmm, I think this is dangerous to do. Let's just make it `6` and ensure we do what's necessary to make `6` a safe number. Letting any value be set in a vacuum / without binding it to other things is a footgun.

Reply:
Understood, I didn't think about the other implications (setting it to 1, etc.). Updated.